ceph.spec.in

```diff
@@ -82,10 +84,10 @@ ExclusiveArch:  x86_64 aarch64
 #################################################################################
 # dependencies that apply across all distro families
 #################################################################################
-Requires:       ceph-osd = %{epoch}:%{version}-%{release}
-Requires:       ceph-mds = %{epoch}:%{version}-%{release}
-Requires:       ceph-mgr = %{epoch}:%{version}-%{release}
-Requires:       ceph-mon = %{epoch}:%{version}-%{release}
+Requires:       ceph-osd = %{?epoch:%{epoch}:}%{version}-%{release}
```

Comment:
okay, i was just googling for how to define a variable in rpm script, and `%define` popped out. if you guys think `%global` is better, i am all for it! =)